### DISCUSSION TITLE
IP POOL : Cannot assign requested address - bind(2) for "IP ADDRESS" port  Support Ref: 0NZ2DDJG

### DISCUSSION THREAD

**Discussion by charlesmudy** [postalserver/postal#3235] [Help with using Postal]
Hello,

Trying to use the IP POOL, I have done the setting on the server for multiple IPs.

When I add multiple IP via the web UI, it works on the first IP but when it try to pick second IP then I get this error

`Cannot assign requested address - bind(2) for "IP ADDRESS" port  Support Ref: 0NZ2DDJG` ... the IP ADDRESS is the actual IP

Any idea to resolve this?

**Comment by willpower232** [comment]
You need to use IP addresses that are directly visible to the server, some server providers do things with private networking so if you run `ip a` on your server, you have to use those specific IP addresses.

**Comment by charlesmudy** [comment]
I just needed to do something around:

`auto lo
iface lo inet loopback
auto eth0
iface eth0 inet dhcp`

Solved.... thanks
